src/BenchmarkExamples/ForLoop/ForLoopBenchmark.cs

```diff
@@ -13,22 +12,24 @@ public class ForLoopBenchmark
         public int N;
 
         public List<int> intList;
+        public int[] intArray;
```

Comment:
Create the array only once in the global setup, otherwise the array creation is measured as part of the benchmark.

Reply:
Also, it was an array of `object` despite being named `intArray` ;)

Reply:
You're right, I was testing to see if int v. object array made any different, and forgot to change it back.